pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -729,7 +758,9 @@ func newCertRotationController(
 			Name:      "control-plane-node-admin-client-cert-key",
 			AdditionalAnnotations: certrotation.AdditionalAnnotations{
 				JiraComponent:                    "kube-apiserver",
-				AutoRegenerateAfterOfflineExpiry: "https://github.com/openshift/cluster-kube-apiserver-operator/pull/1631,'[Conformance][sig-api-machinery][Feature:APIServer] local kubeconfig \"control-plane-node.kubeconfig\" should be present in all kube-apiserver containers [Suite:openshift/conformance/parallel/minimal]'",
+				Description:                      "kube-controller-manager and kube-scheduler client certificates.",
```

Comment:
from the PR description:
```
This PR also updates conflicting descriptions to kube-control-plane-signer secret to make sure controllers don't hotloop erasing each other changes
```

does adding a description prevents controllers from erasing changes added by the others controllers?